### PR TITLE
9.1 Muharrem Order App with View Model

### PR DESCRIPTION
**Pull request by Muharrem-sdet**
9.1 Implement View Model (Order App Kotlin Version)

### CODE REVIEW

File: OrderAppKotlinVersion/app/src/main/java/com/example/orderappkotlinversion/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        text_view_amount.text = orderViewModel.count.toString()
+        text_view_receipt.text = orderViewModel.text_view_receipt_label
+
+        button_increase.setOnClickListener {
+            orderViewModel.count++
+            text_view_amount.text = orderViewModel.count.toString()
+        }
+
+        button_decrease.setOnClickListener {
+            if (orderViewModel.count > 0) {
```
Comment (Kryvovyaz) [inline OrderAppKotlinVersion/app/src/main/java/com/example/orderappkotlinversion/MainActivity.kt:32]: you still keep logic in main

Comment (dzemal5) [reply OrderAppKotlinVersion/app/src/main/java/com/example/orderappkotlinversion/MainActivity.kt:32]: get your count in View Model

Comment (Muharrem-sdet) [reply OrderAppKotlinVersion/app/src/main/java/com/example/orderappkotlinversion/MainActivity.kt:32]: count is already in view model

**Comment by oliver-basoglu** [comment]
Look's great

**Pull request merged**